apps/emqx_auth_http/priv/emqx_auth_http.schema

```diff
@@ -110,8 +110,8 @@ end}.
 ]}.
 
 {mapping, "auth.http.enable_pipelining", "emqx_auth_http.enable_pipelining", [
-  {default, true},
-  {datatype, {enum, [true, false]}}
+  {default, 100},
+  {datatype, integer}
 ]}.
```

Comment:
we can't make such change in 4.3
have to be backward compatible.
change it to a string field and add a translation.